[CO-3243] Added documentation for scheduled bulk users merge - MERGE 1/27 (#8825)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/user_profiles/duplicate_users.md

If you're satisfied with the results of your preview, return to the **Manage Audience** page and select **Merge all duplicates**.

{% alert warning %}
Duplicate user profiles cannot be recovered after merging.
{% endalert %}

## Scheduled merging

> Similar to rules-based merging, scheduled merging allows you to automate the merging of user profiles on a daily basis using preconfigured rules.

Reviewer: ```suggestion
Similar to rules-based merging, scheduled merging allows you to automate the merging of user profiles on a daily basis using preconfigured rules.

{% alert important %}
Scheduled merging is in early access. Contact your Braze account manager if you're interested in participating in this early access.
{% endalert %}
```

![The "Manage Audience" page with "schedule" button.]({% image_buster /assets/img/audience_management/duplicate_users/bulk_merging/select_scheduled_merge_rules.png %})

After the feature is turned on, Braze will automatically assign a timeslot to perform the merge process daily. You can turn off scheduled merging at any time. Braze will notify the admins of your workspace 24 hours before the scheduled merge occurs, providing a reminder and time to review the configuration.

> 

Reviewer: ```suggestion
```

> Similar to rules-based merging, the scheduled users merging feature allows you to automate the merging of user profiles on a daily basis using pre-configured rules.
> 
> ![The "Manage Audience" page with "schedule" button.]({% image_buster /assets/img/audience_management/duplicate_users/bulk_merging/select_scheduled_merge_rules.png %})
> 
> Once the feature is enabled, Braze will automatically assign a timeslot to perform the merge process daily. You can disable schedule merging at any time if needed. Braze will notify the admins of your app group 24 hours before the scheduled merge occurs, providing a reminder and time to review the configuration.

Reviewer: ```suggestion
Similar to rules-based merging, scheduled merging allows you to automate the merging of user profiles on a daily basis using preconfigured rules.

![The "Manage Audience" page with "schedule" button.]({% image_buster /assets/img/audience_management/duplicate_users/bulk_merging/select_scheduled_merge_rules.png %})

After the feature is turned on, Braze will automatically assign a timeslot to perform the merge process daily. You can turn off scheduled merging at any time. Braze will notify the admins of your workspace 24 hours before the scheduled merge occurs, providing a reminder and time to review the configuration.
```
